backend/src/types/index.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs'
+import path from 'path'
+import { mergeTypes } from 'merge-graphql-schemas'
+
+const findGqlFiles = dir => {
+  var results = []
+  var list = fs.readdirSync(dir)
+  list.forEach(file => {
+    file = path.join(dir, file).toString('utf-8')
+    var stat = fs.statSync(file)
+    if (stat && stat.isDirectory()) {
+      // Recurse into a subdirectory
+      results = results.concat(findGqlFiles(file))
+    } else {
+      if (path.extname(file) === '.gql') {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-31T13:05:23Z" title="Friday, May 31st 2019, 3:05:23 pm +02:00">May 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,30 @@
+import fs from 'fs'
+import path from 'path'
+import { mergeTypes } from 'merge-graphql-schemas'
+
+const findGqlFiles = dir => {
+  var results = []
+  var list = fs.readdirSync(dir)
+  list.forEach(file => {
+    file = path.join(dir, file).toString('utf-8')
+    var stat = fs.statSync(file)
+    if (stat && stat.isDirectory()) {
+      // Recurse into a subdirectory
+      results = results.concat(findGqlFiles(file))
+    } else {
+      if (file.split('.').pop() === 'gql') {
```


I guess there is a cleaner way to check for a file extension.